specification/redisenterprise/resource-manager/Microsoft.Cache/stable/2021-03-01/examples/OperationsStatusGet.json

```diff
@@ -0,0 +1,19 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
+    "location": "West US",
+    "operationId": "testoperationid",
+    "api-version": "2021-03-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subid/providers/Microsoft.Cache/locations/westus/operationStatus/testoperationid",
```

Comment:
Shouldn't this be operationsStatus?

Reply:
Yes

Reply:
Good catch! Do I re-open this PR with this fix or my other PR: https://github.com/Azure/azure-rest-api-specs/pull/12943 ?

Reply:
I'd say a tiny separate PR.

Reply:
Okay great. here it is: https://github.com/Azure/azure-rest-api-specs/pull/12967